File: nodejs/MongoDB.js
var MongoClient = require("mongodb").MongoClient;

var hostname = "<$hostname>";
var database = "<$database>";
var port = "<$port>";
var username = "<$user>";
var password = "<$password>";

var url =
  "mongodb://" +
  username +
  ":" +
  password +
  "@" +
  hostname +
  ":" +
  port +
  "/" +
  database;

MongoClient.connect(url, function (err, db) {
  if (err) throw err;
  console.log("Connected to database!");
});

File: go/MongoDB.go
package main

import (
	"context"
	"fmt"
	"log"

	"go.mongodb.org/mongo-driver/mongo"
	"go.mongodb.org/mongo-driver/mongo/options"
)

const (
	host     = "<$hostname>"
	port     = "<$port>"
	user     = "<$user>"
	password = "<$password>"
	dbname   = "<$database>"
)

func main() {
	// connection string
	uri := fmt.Sprintf("mongodb://%s:%s@%s:%s/%s", user, password, host, port, dbname)

	clientOptions := options.Client().ApplyURI("mongodb://localhost:27017")

	// Connect to MongoDB
	client, err := mongo.Connect(context.TODO(), clientOptions)

	if err != nil {
		log.Fatal(err)
	}

	// Check the connection
	err = client.Ping(context.TODO(), nil)

	if err != nil {
		log.Fatal(err)
	}

	fmt.Println("Connected to MongoDB!")

	err = client.Disconnect(context.TODO())

	if err != nil {
		log.Fatal(err)
	}
	fmt.Println("Connection to MongoDB closed.")
}
